static/src/components/ImageDisplay.vue

```diff
@@ -1,7 +1,16 @@
-<script setup lang="ts">
-defineProps<{
-  src: string;
-}>();
+<script lang="ts">
+import { defineComponent } from 'vue'
+/**
+ * Displays an image specified by `src`
+ */
+export default defineComponent({
+  props: {
+    /**
+     * Source from where the image should be loaded.
+     */
+    src: { type: String, required: true }
+  },
+})
```

Comment:
Interesting, may I ask what's the benefit of this method over the old one?

Reply:
In theory, either way works the same. Vue has two ways how to define a component with its properties and methods and so on. One is called the Composition API and the other one is the Options API. Both lead to the exact same component but the options api is told to be easier for beginners. Since that sounded good, we used the options api in the AnnotationButton and swapped it out in the ImageDisplay for consistency.
I've not yet dived into that topic very deeply, but here are the links I used for the information on this topic:
https://vuejs.org/guide/introduction.html#api-styles
https://vuejs.org/guide/typescript/composition-api.html
https://vuejs.org/guide/typescript/options-api.html